dpypelines/pipeline/dataset_ingress_v1.py

```diff
@@ -1,22 +1,92 @@
 from pathlib import Path
 
+from dpytools.stores.directory.local import LocalDirectoryStore
+from dpypelines.pipeline.shared import notification as notify
+from dpypelines.pipeline.shared.pipelineconfig import matching
+from dpypelines.pipeline.functions.config import get_pipeline_identifier_from_config
+from pipelines.pipeline.shared import message
 
-# IMPORTANT
-# We probably want to break these stages out a bit
-# to help with reusibility
-def dataset_ingress_v1(files_dir: Path):
+def dataset_ingress_v1(files_dir: str) -> None:
```

Comment:
dont hint `-> None`, callables that dont have a return statement implicitly return None, its not something you need to hint (otherwise you'd see it everywhere).

The `->` is for things that you are a programmer are choosing to return, its a hint to capture the intent of the programmer who wrote the code.